object_tracker/object_tracker_test.go

```diff
@@ -340,3 +341,37 @@ func TestTracker(t *testing.T) {
 	test.That(t, len(currDetections), test.ShouldEqual, 1)
 	checkLabel(t, currDetections[0], LabelDet0)
 }
+
+func TestInvalidCameraNamesError(t *testing.T) {
+	ctx := context.Background()
+	fakeTracker := &myTracker{
+		camName: "test",
+		cancelContext: ctx,
+	}
+
+	invalidName := "not-camera"
+	invalidNameErrorMessage := "Camera name given to method, not-camera is not the same as configured camera test"
+
+	// Test invalid camera name in DetectionsFromCamera
+	_, err := fakeTracker.DetectionsFromCamera(ctx, invalidName, make(map[string]interface{}))
```

Comment:
And maybe make this empty map once, call it emptyMap, and reuse it.  This one I don't _actually_ care about.  Just a thought.